pkg/encryption/password.go

```diff
@@ -25,19 +24,19 @@ func sha256hmac(key, data []byte) ([]byte, error) {
 
 // DeriveRootKey derives a root key for some path using the salt for the bucket and
 // a password from the user. See the password key derivation design doc.
-func DeriveRootKey(password, salt []byte, path storj.Path) (*storj.Key, error) {
+func DeriveRootKey(password, salt []byte, argon2Threads uint8) (*storj.Key, error) {
```

Comment:
Removing the path changes the design (https://github.com/storj/storj/blob/master/docs/blueprints/password-key-derivation.md#design)

While it's not used, it exists to allow different root encryption keys at different paths inside of a bucket.

Reply:
okay nbd to bring it back. i wrongly assumed it was refactor cruft